spec/models/bookmark_search_form_spec.rb

```diff
@@ -10,4 +10,84 @@
       expect(bsf.options).to include(show_private: false)
     end
   end
+
+  describe "bookmarkable_search_results" do
+    describe "sorting" do
+      let(:tag) { create(:canonical_fandom) }
+
+      let!(:work1) do
+        Delorean.time_travel_to 40.minutes.ago do
```

Comment:
Nice use of `time_travel_to` to create items at specific relative dates rather than our usual approach of hard-coding dates that come back to bite us. *takes notes*